executor/executor.go

```diff
@@ -296,6 +297,61 @@ type ShowDDLJobsExec struct {
 	jobs   []*model.Job
 }
 
+// ShowDDLJobQueriesExec represents a show DDL job queries executor.
+// The `admin show ddl jos` command and `admin show ddl job queries job_id` use together.
+// The command query has completed job history sql max limit 10 .
```

Comment:
Can not understand this.

Reply:
`admin show ddl jobs`
```
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
| JOBS                                                                                                                                                                                              | STATE  |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
| ID:32, Type:create table, State:synced, SchemaState:public, SchemaID:1, TableID:31, RowCount:0, ArgLen:0, start time: 2018-03-13 14:34:27.314 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:30, Type:create table, State:synced, SchemaState:public, SchemaID:1, TableID:29, RowCount:0, ArgLen:0, start time: 2018-03-13 14:34:14.123 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:28, Type:create table, State:synced, SchemaState:public, SchemaID:1, TableID:27, RowCount:0, ArgLen:0, start time: 2018-03-13 14:33:50.999 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:26, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:25, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:07.551 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:24, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:23, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:07.423 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:22, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:21, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:07.304 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:20, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:19, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:07.123 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:18, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:17, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:07.007 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:16, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:15, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:06.896 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
| ID:14, Type:create table, State:synced, SchemaState:public, SchemaID:3, TableID:13, RowCount:0, ArgLen:0, start time: 2018-03-12 15:15:06.783 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:0 | synced |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------+
```
`admin show ddl job queries 26;`
```
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| QUERY                                                                                                                                                                                                                                                                                                                                                                                                  |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| CREATE TABLE IF NOT EXISTS mysql.gc_delete_range (
		job_id BIGINT NOT NULL COMMENT "the DDL job ID",
		element_id BIGINT NOT NULL COMMENT "the schema element ID",
		start_key VARCHAR(255) NOT NULL COMMENT "encoded in hex",
		end_key VARCHAR(255) NOT NULL COMMENT "encoded in hex",
		ts BIGINT NOT NULL COMMENT "timestamp in int64",
		UNIQUE KEY (element_id),
		KEY (job_id, element_id)
	); |
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
The comment is not clear.